ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/stores/SandboxIndexAndWriteService.scala

```diff
@@ -385,7 +396,7 @@ abstract class LedgerBackedIndexService(
   override def getParticipantId(): Future[ParticipantId] =
     // In the case of the sandbox, there is only one participant node
     // TODO: Make the participant ID configurable
-    Future.successful(ParticipantId(ledger.ledgerId.unwrap))
+    Future.successful(Ref.LedgerString.assertFromString(ledger.ledgerId.unwrap))
```

Comment:
We can return the participant ID here, no?

Reply:
indeed, thanks for spotting this; that was the very reason for this entire storm in the glass

Reply:
Done